platforms/posix/src/px4_layer/drv_hrt.cpp

```diff
@@ -68,7 +68,7 @@ static int32_t dsp_offset = 0;
 #endif
 
 #if defined(ENABLE_LOCKSTEP_SCHEDULER)
-static LockstepScheduler lockstep_scheduler;
+static LockstepScheduler *lockstep_scheduler = new LockstepScheduler();
```

Comment:
Not sure what to think about using new in static initialization.

Reply:
I'm not saying I like it and it's a fair concern - but I also want to avoid adding null-checks.
We can also tell the LockstepScheduler to not do any cleanup.